From 31c3b9c95e2b14189f2bc9f2bd886834114da81d Mon Sep 17 00:00:00 2001
From: Justin Ruggles <justin.ruggles@gmail.com>
Date: Sat, 27 Oct 2012 01:22:07 -0400
Subject: [PATCH 185/278] FATE: rename FLAC tests from flac-* to flac-16-*

This will distinguish FLAC 16-bit tests from FLAC tests with other bit
depths.
---
 tests/fate/flac.mak |   25 +++++++++++++------------
 1 file changed, 13 insertions(+), 12 deletions(-)

diff --git a/tests/fate/flac.mak b/tests/fate/flac.mak
index dc49928..f7ac342 100644
--- a/tests/fate/flac.mak
+++ b/tests/fate/flac.mak
@@ -1,17 +1,18 @@
-FATE_FLAC += fate-flac-chmode-indep                                     \
-             fate-flac-chmode-left_side                                 \
-             fate-flac-chmode-mid_side                                  \
-             fate-flac-chmode-right_side                                \
-             fate-flac-fixed                                            \
-             fate-flac-lpc-cholesky                                     \
-             fate-flac-lpc-levinson                                     \
+FATE_FLAC += fate-flac-16-chmode-indep                                  \
+             fate-flac-16-chmode-left_side                              \
+             fate-flac-16-chmode-mid_side                               \
+             fate-flac-16-chmode-right_side                             \
+             fate-flac-16-fixed                                         \
+             fate-flac-16-lpc-cholesky                                  \
+             fate-flac-16-lpc-levinson                                  \
 
-fate-flac-chmode-%: OPTS = -ch_mode $(@:fate-flac-chmode-%=%)
-fate-flac-fixed: OPTS = -lpc_type fixed
-fate-flac-lpc-%: OPTS = -lpc_type $(@:fate-flac-lpc-%=%)
+fate-flac-16-chmode-%: OPTS = -ch_mode $(@:fate-flac-16-chmode-%=%)
+fate-flac-16-fixed:    OPTS = -lpc_type fixed
+fate-flac-16-lpc-%:    OPTS = -lpc_type $(@:fate-flac-16-lpc-%=%)
+
+fate-flac-16-%: REF = $(SAMPLES)/audio-reference/luckynight_2ch_44kHz_s16.wav
+fate-flac-16-%: CMD = enc_dec_pcm flac wav s16le $(REF) -c flac $(OPTS)
 
-fate-flac-%: REF = $(SAMPLES)/audio-reference/luckynight_2ch_44kHz_s16.wav
-fate-flac-%: CMD = enc_dec_pcm flac wav s16le $(REF) -c flac $(OPTS)
 fate-flac-%: CMP = oneoff
 fate-flac-%: FUZZ = 0
 
-- 
1.7.9.5

